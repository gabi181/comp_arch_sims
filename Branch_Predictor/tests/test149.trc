16 3 26 1 local_history global_tables using_share_mid
0x5E9F0FDC T 0x1D55B7DC
0x5E9F0FDC T 0x1D55B7DC
0x31FFBC50 N 0x06C08180
0x5E9F0FDC T 0x1D55B7DC
0x3E42E828 T 0x785750AC
0x3D8A0EC0 N 0x5C7B3178
0x3D8A0EC0 N 0x5C7B3178
0x5E9F0FDC N 0x1D55B7DC
0x3E42E828 T 0x785750AC
0x5E9F0FDC T 0x1D55B7DC
0x3D2D78C4 N 0x7E53924C
0x5E9F0FDC T 0x1D55B7DC
0x5E9F0FDC N 0x1D55B7DC
0x3D8A0EC0 N 0x5C7B3178
0x3E42E828 T 0x785750AC
0x3E42E828 N 0x785750AC
0x3D2D78C4 T 0x7E53924C
0x3D2D78C4 T 0x7E53924C
0x3D8A0EC0 T 0x5C7B3178
0x5E9F0FDC T 0x1D55B7DC
0x3E42E828 N 0x785750AC
