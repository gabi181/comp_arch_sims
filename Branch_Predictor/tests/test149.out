0x5e9f0fdc N 0x5e9f0fe0
0x5e9f0fdc N 0x5e9f0fe0
0x31ffbc50 N 0x31ffbc54
0x5e9f0fdc N 0x5e9f0fe0
0x3e42e828 N 0x3e42e82c
0x3d8a0ec0 N 0x3d8a0ec4
0x3d8a0ec0 N 0x3d8a0ec4
0x5e9f0fdc N 0x5e9f0fe0
0x3e42e828 N 0x3e42e82c
0x5e9f0fdc N 0x5e9f0fe0
0x3d2d78c4 N 0x3d2d78c8
0x5e9f0fdc N 0x5e9f0fe0
0x5e9f0fdc T 0x1d55b7dc
0x3d8a0ec0 N 0x3d8a0ec4
0x3e42e828 T 0x785750ac
0x3e42e828 N 0x3e42e82c
0x3d2d78c4 N 0x3d2d78c8
0x3d2d78c4 N 0x3d2d78c8
0x3d8a0ec0 N 0x3d8a0ec4
0x5e9f0fdc T 0x1d55b7dc
0x3e42e828 T 0x785750ac
flush_num: 12, br_num: 21, size: 976b
